### PR TITLE
Update translation-ko.json

### PR DESCRIPTION
**Pull request by JoYongpil**
update Korean translation

### CODE REVIEW

**Comment by JoYongpil** [comment]
surfly.com uses PDFTron. they updated to version 8.2.0. however korean translation isn't applied. please let me know which version is updated korean translation be available in.
IMG_0040
IMG_0041

**Comment by bollain** [comment]
Hi @JoYongpil,

Thanks for your PR. Are these translations in addition to the ones you contributed in this PR?
https://github.com/PDFTron/webviewer-ui/commit/d9f5178affc2e15c09f3213fcbbbdaafdb0bf49e

Or is there another issue here?

File: i18n/translation-ko.json [hunk 1]
```diff
@@ -71,13 +71,13 @@
     "insertBlankPageBelow": "아래에 빈 페이지 삽입",
     "pageManipulation": "페이지 편집",
     "replace": "바꾸기",
-    "showLess": "덜 보여",
-    "showMore": "...더",
+    "showLess": "간단히 보기",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:74]: change translation

File: i18n/translation-ko.json [hunk 2]
```diff
@@ -71,13 +71,13 @@
...
+    "showMore": "더 보기",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:75]: change translation

File: i18n/translation-ko.json [hunk 3]
```diff
@@ -71,13 +71,13 @@
...
     "chooseFile": "파일 선택",
     "selectYourOption": "옵션을 선택하세요",
-    "open": "열려있는",
-    "deselectAll": "모두 선택 해제",
-    "select": "선택하다",
+    "open": "열기",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:78]: change translation

File: i18n/translation-ko.json [hunk 4]
```diff
@@ -71,13 +71,13 @@
...
+    "deselectAll": "모두 선택해제",
+    "select": "선택",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:80]: change translation

File: i18n/translation-ko.json [hunk 5]
```diff
@@ -105,8 +105,8 @@
     "polygon": "다각형",
     "polygonCloud": "구름",
     "polyline": "꺾은선",
-    "rectangle": "직사각형",
-    "redact": "민감 정보 삭제",
+    "rectangle": "사각형",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:108]: change translation

File: i18n/translation-ko.json [hunk 6]
```diff
@@ -220,33 +220,33 @@
...
     "selectPageToReplace": "문서에서 교체할 페이지를 선택합니다.",
     "embeddedFiles": "포함된 파일",
     "pageNum": "페이지 번호"
   },
   "option": {
     "type": {
-      "caret": "부족하다",
-      "custom": "관습",
-      "ellipse": "타원",
+      "caret": "커서",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:231]: change translation

File: i18n/translation-ko.json [hunk 7]
```diff
@@ -220,33 +220,33 @@
...
+      "custom": "사용자 지정",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:232]: change translation

File: i18n/translation-ko.json [hunk 8]
```diff
@@ -220,33 +220,33 @@
...
+      "ellipse": "타원형",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:233]: change translation

File: i18n/translation-ko.json [hunk 9]
```diff
@@ -220,33 +220,33 @@
...
       "fileattachment": "파일 첨부",
-      "freehand": "자유 재량",
-      "callout": "호출",
-      "freetext": "무료 문자",
+      "freehand": "그리기",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:235]: change translation

File: i18n/translation-ko.json [hunk 10]
```diff
@@ -220,33 +220,33 @@
...
+      "callout": "설명선",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:236]: change translation

File: i18n/translation-ko.json [hunk 11]
```diff
@@ -220,33 +220,33 @@
...
+      "freetext": "텍스트 상자",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:237]: change translation

File: i18n/translation-ko.json [hunk 12]
```diff
@@ -220,33 +220,33 @@
...
       "line": "선",
       "polygon": "다각형",
-      "polyline": "폴리라인",
-      "rectangle": "직사각형",
-      "redact": "수정",
+      "polyline": "꺾은선",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:240]: change translation

File: i18n/translation-ko.json [hunk 13]
```diff
@@ -220,33 +220,33 @@
...
+      "rectangle": "사각형",
+      "redact": "민감정보 삭제",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:242]: change translation

File: i18n/translation-ko.json [hunk 14]
```diff
@@ -220,33 +220,33 @@
...
       "signature": "서명",
-      "stamp": "우표",
-      "stickyNote": "스티커 메모",
-      "highlight": "가장 밝은 부분",
-      "strikeout": "삼진",
+      "stamp": "이미지",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:244]: change translation

File: i18n/translation-ko.json [hunk 15]
```diff
@@ -220,33 +220,33 @@
...
+      "stickyNote": "코멘트",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:245]: change translation

File: i18n/translation-ko.json [hunk 16]
```diff
@@ -220,33 +220,33 @@
...
+      "highlight": "하이라이트",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:246]: change translation

File: i18n/translation-ko.json [hunk 17]
```diff
@@ -220,33 +220,33 @@
...
+      "strikeout": "취소선",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:247]: change translation

File: i18n/translation-ko.json [hunk 18]
```diff
@@ -220,33 +220,33 @@
...
       "underline": "밑줄",
-      "squiggly": "구불구불한",
+      "squiggly": "물결선",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:249]: change translation

File: i18n/translation-ko.json [hunk 19]
```diff
@@ -258,8 +258,8 @@
       "author": "작성자",
       "type": "유형",
       "color": "색상",
-      "createDate": "만든 날짜",
-      "modifiedDate": "수정 날짜"
+      "createDate": "작성일",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:261]: change translation

File: i18n/translation-ko.json [hunk 20]
```diff
@@ -258,8 +258,8 @@
...
+      "modifiedDate": "수정일"
```
Comment (JoYongpil) [inline i18n/translation-ko.json:262]: change translation

File: i18n/translation-ko.json [hunk 21]
```diff
@@ -448,7 +448,7 @@
       "strikeout": "취소선",
       "alignLeft": "텍스트 왼쪽 정렬",
       "alignRight": "텍스트 오른쪽 정렬",
-      "alignCenter": "텍스트 정렬 중심",
+      "alignCenter": "텍스트 가운데 정렬",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:451]: change translation

File: i18n/translation-ko.json [hunk 22]
```diff
@@ -460,8 +460,8 @@
       "timestampText": "타임 스탬프 텍스트 :",
       "stampColor": "스탬프 색상 :",
       "Username": "사용자 이름",
-      "Date": "데이트",
-      "Time": "시각"
+      "Date": "날짜",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:463]: change translation

File: i18n/translation-ko.json [hunk 23]
```diff
@@ -460,8 +460,8 @@
...
+      "Time": "시간"
```
Comment (JoYongpil) [inline i18n/translation-ko.json:464]: change translation

File: i18n/translation-ko.json [hunk 24]
```diff
@@ -472,7 +472,7 @@
     },
     "extractPage": {
       "title": "페이지 다운로드",
-      "message": "선택한 페이지를 다운로드하시겠습니까?",
+      "message": "선택한 페이지를 다운로드 하시겠습니까?",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:475]: word spacing

File: i18n/translation-ko.json [hunk 25]
```diff
@@ -220,33 +220,33 @@
...
+      "freehand": "그리기",
+      "callout": "설명선",
+      "freetext": "텍스트 상자",
       "line": "선",
       "polygon": "다각형",
-      "polyline": "폴리라인",
-      "rectangle": "직사각형",
-      "redact": "수정",
+      "polyline": "꺾은선",
+      "rectangle": "사각형",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:241]: change translation

File: i18n/translation-ko.json [hunk 26]
```diff
@@ -220,33 +220,33 @@
     "selectHowToLoadFile": "문서 로드 방법 선택",
     "openFileByUrl": "URL로 파일 열기:",
     "enterUrlHere": "여기에 URL을 입력하세요.",
-    "openLocalFile": "로컬 파일 열기:",
+    "openLocalFile": "로컬파일 열기:",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:223]: word spacing

File: i18n/translation-ko.json [hunk 27]
```diff
@@ -168,7 +168,7 @@
     "layersPanel": "레이어",
     "thumbnailsPanel": "미리보기",
     "toolsButton": "도구",
-    "redaction": "민감 정보 삭제",
+    "redaction": "민감정보 삭제",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:171]: word spacing

File: i18n/translation-ko.json [hunk 28]
```diff
@@ -105,8 +105,8 @@
     "polygon": "다각형",
     "polygonCloud": "구름",
     "polyline": "꺾은선",
-    "rectangle": "직사각형",
-    "redact": "민감 정보 삭제",
+    "rectangle": "사각형",
+    "redact": "민감정보 삭제",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:109]: word spacing

File: i18n/translation-ko.json [hunk 29]
```diff
@@ -71,13 +71,13 @@
...
-    "showMore": "...더",
+    "showLess": "간단히 보기",
+    "showMore": "더 보기",
     "chooseFile": "파일 선택",
     "selectYourOption": "옵션을 선택하세요",
-    "open": "열려있는",
-    "deselectAll": "모두 선택 해제",
-    "select": "선택하다",
+    "open": "열기",
+    "deselectAll": "모두 선택해제",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:79]: word spacing

File: i18n/translation-ko.json [hunk 30]
```diff
@@ -52,9 +52,9 @@
     "type": "입력",
     "upload": "업로드",
     "link": "링크",
-    "darkMode": "다크 모드",
-    "lightMode": "라이트 모드",
-    "fileAttachmentDownload": "첨부 파일 다운로드",
+    "darkMode": "다크모드",
+    "lightMode": "라이트모드",
+    "fileAttachmentDownload": "첨부파일 다운로드",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:57]: word spacing

File: i18n/translation-ko.json [hunk 31]
```diff
@@ -52,9 +52,9 @@
     "type": "입력",
     "upload": "업로드",
     "link": "링크",
-    "darkMode": "다크 모드",
-    "lightMode": "라이트 모드",
-    "fileAttachmentDownload": "첨부 파일 다운로드",
+    "darkMode": "다크모드",
+    "lightMode": "라이트모드",
```
Comment (JoYongpil) [inline i18n/translation-ko.json:56]: word spacing

**Comment by JoYongpil** [comment]
as it was updated to version 8.2.0, i've found new lines with mistranslation. please check comments.

**Comment by JoYongpil** [comment]
should i move this comment to the issues tap? 

> surfly.com uses PDFTron. they updated to version 8.2.0. however korean translation isn't applied. please let me know which version is updated korean translation be available in.
> IMG_0040
> IMG_0041

**Comment by bollain** [comment]
Thanks for the clarification @JoYongpil! It is better to track the issue separately, you can submit a ticket [here](https://www.pdftron.com/form/request/) or alternatively file an issue with clear reproduction steps. For the translations, we will review them internally and let you know if there is any feedback.

**Review by bollain — APPROVED** [review]
Translations have been reviewed internally and are ready to be merged. Thanks again for your contribution @JoYongpil!! If you are not building the UI project yourself, you will find an updated version of 8.2 from our nightly builds in 1-2 days. 
https://www.pdftron.com/nightly/#stable/

**Pull request merged**